src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -0,0 +1,221 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form, Icon, StatefulButton, useCheckboxSetValues, useToggle,
+} from '@edx/paragon';
+import { SpinnerSimple, CheckCircle, Cancel } from '@edx/paragon/icons';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import TextEditor from '../text-editor/TextEditor';
+import { postBulkEmail } from './api';
+import BulkEmailRecepient from './BulkEmailRecepient';
+import TaskAlertModal from './TaskAlertModal';
+import useTimeout from '../../../utils/useTimeout';
+
+export const FORM_SUBMIT_STATES = {
+  DEFAULT: 'default',
+  PENDING: 'pending',
+  COMPLETE: 'complete',
+  COMPLETED_DEFAULT: 'completed_default',
+  ERROR: 'error',
+};
+
+export default function BulkEmailForm(props) {
+  const { courseId } = props;
+  const [subject, setSubject] = useState('');
+  const [emailFormStatus, setEmailFormStatus] = useState(FORM_SUBMIT_STATES.DEFAULT);
+  const [emailformValidation, setEmailFormValidation] = useState({
```

Comment:
mega-nit: should `emailformValidation` be `emailFormValidation?` to match the camel casing of `setEmailFormValidation`?

Reply:
Yes you are correct! fixed it.